.github/workflows/commit-message-check.yml

```
@@ -0,0 +1,26 @@
+name: Commit Message Check
+
+on: [pull_request, push]
+
+jobs:
+  check-commit-message:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)